File: frontend/src/app/views/profile/room.ts
export class Room {

  public ID: number;
  public place: number;
  public clas: number;
  public booked: boolean;
  public price: number;

  constructor(ID: number, place: number, clas: number, booked: boolean, price: number) {
    this.ID = ID;
    this.place = place;
    this.clas = clas;
    this.booked = booked;
    this.price = price;
  }
}

File: servlet/src/main/java/com/example/entities/Room.java
package com.example.entities;

import com.fasterxml.jackson.annotation.JsonPropertyOrder;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@JsonPropertyOrder({"ID", "place", "clas", "booked","price"})
public class Room {
    private int ID;
    private int place;
    private int clas;
    private boolean booked;
    private float price;
}
